openid-4-verifiable-presentations-1_0.md

```diff
@@ -2362,7 +2362,7 @@ If `require_cryptographic_holder_binding` is set to `true` in the Credential Que
 The following is a W3C Verifiable Credentials specific parameter in the `meta` parameter in a Credential Query as defined in (#credential_query):
 
 `type_values`:
-: REQUIRED. An array of string arrays that specifies the fully expanded types (IRIs) after the `@context` was applied that the Verifier accepts to be presented in the Presentation. Each of the top-level arrays specifies one alternative to match the `type` values of the Verifiable Credential against. Each inner array specifies a set of fully expanded types that MUST be present in the `type` property of the Verifiable Credential, regardless of order or the presence of additional types. 
+: REQUIRED. An array of string arrays that specifies the fully expanded types (IRIs) after the `@context` was applied that the Verifier accepts to be presented in the Presentation. If a type value was not defined by any `@context`, its value stays the same. This works fine in practice, even if it's not strictly JSON-LD compliant. Implementations can use alternative mechanisms that produce the same outcome as applying the `@context`, provided that the result is equivalent. Each of the top-level arrays specifies one alternative to match the `type` values of the Verifiable Credential against. Each inner array specifies a set of fully expanded types that MUST be present in the `type` property of the Verifiable Credential, regardless of order or the presence of additional types. 
```

Comment:
few clarification questions:

> If a type value was not defined by any `@context`, its value stays the same. 

what value?

> This works fine in practice, even if it's not strictly JSON-LD compliant. 

i would just say `This works in practice` to make it a bit more formal and avoid talking about JSON-LD compliance..?

> Each of the top-level arrays specifies one alternative to match the `type` values of the Verifiable Credential against. 

which top level arrays..?

Reply:
> what value?

I believe what @awoie is referring to is the original type value in its non-expanded JSON-LD form

> i would just say This works in practice to make it a bit more formal and avoid talking about JSON-LD compliance..?

I would go one step further and not even say "works in practise" the fact we have defined this in the spec means implicitly it works.